src/granite/base.cr

```diff
@@ -70,5 +93,10 @@ abstract class Granite::Base
 
     disable_granite_docs? def initialize
     end
+
```

Comment:
Just a side note, I wonder if there is a way to ensure that these are ALWAYS performed first, that way if someone writes a callback that fails, we don't miss updating the last write time

Reply:
We can worry about this once it's fully fleshed out, I bet there will be other aspects that crop up that might resolve this on it's own